geopattern/geopattern-tests.ts

```diff
@@ -1,4 +1,7 @@
 /// <reference path="index.d.ts" />
+/// <reference path="../jquery/index.d.ts" />
```

Comment:
Neither is needed, `index.d.ts` is brought into the compilation by `tsconfig.json`, and it references `jquery`.